### PR TITLE
Reworking Docs

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Describe the problem or feature in addition to a link to the issues.

**Approach**
How does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [ ] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by nemchik** [comment]
Didn't mean to put this off, sorry. Can you resolve conflicts and update anything related to volumes with the new `/storage` info?

File: docs/basics/env-var-info.md [hunk 1]
```diff
@@ -118,22 +118,23 @@ This directory will be mounted under `/tv` inside any container that is used to
 * Default value: `1000` because this is the most common default on supported OS
 
 Description:
-This value can be obtained by using `id $user`.
+This value can be obtained by using `id -g $user`.
```
Comment (nemchik) [inline docs/basics/env-var-info.md:134]:
```suggestion
This value can be obtained by using `id -g $USER`.
```

File: docs/basics/env-var-info.md [hunk 2]
```diff
@@ -118,22 +118,23 @@ This directory will be mounted under `/tv` inside any container that is used to
...
 
 ## PUID
 
 * System Detected value: Detects the ID of your user
 * Default value: `1000` because this is the most common default on supported OS
 
 Description:
-This value can be obtained by using `id $user`.
+This value can be obtained by using `id -u $user`.
```
Comment (nemchik) [inline docs/basics/env-var-info.md:142]:
```suggestion
This value can be obtained by using `id -u $USER`.
```

**Review by nemchik — CHANGES_REQUESTED** [review]

File: docs/basics/env-var-info.md (hunk 2 above)
Comment (angrycuban13) [reply docs/basics/env-var-info.md]: 43430e1

**Comment by angrycuban13** [comment]
I'll just do another PR because I somehow messed this one up lol.

**Pull request closed without merging**